log4j-core/src/main/java/org/apache/logging/log4j/core/appender/rolling/DefaultRolloverStrategy.java

```diff
@@ -94,8 +285,10 @@
      * @param stopCustomActionsOnError whether to stop executing asynchronous actions if an error occurs
      * @param config The Configuration.
      * @return A DefaultRolloverStrategy.
+     * @deprecated Since log4j-2.8.3 Usage of Builder API is preferable
```

Comment:
The next release will be 2.9.0. We still have 2.8.3-SNAPSHOT in the POM files as a quirk of our development process. If you look at the changes.xml file, you will see "2.9.0" as the next release :-)